consensus/src/test/kotlin/maru/consensus/qbft/EagerQbftBlockCreatorTest.kt

```diff
@@ -123,6 +127,9 @@ class EagerQbftBlockCreatorTest {
     whenever(
       validatorProvider.getValidatorsAfterBlock(0u),
     ).thenReturn(completedFuture(validatorSet))
+    whenever(
+      prevRandaoProvider.calculateNextPrevRandao(any(), any()),
```

Comment:
`prevRandaoProvider` is a functional interface, so you can just instantiate it instead of mocking it